scalableminds/webknossos
# Team edit modal

### Steps to test:
- Go to teams list 
- Click on `Add/Remove Users`
- Wait for modal to load (demo only)
- click into modal
- remove and add users
- make sure changes are updating the next time the dropdown is opening

### Issues:
- fixes #7035 

------
(Please delete unneeded items, merge only when none are left open)
~~-[ ] Updated [documentation](../blob/master/docs) if applicable~~
